SignalMessaging/attachments/MediaMessageView.swift

```diff
@@ -456,6 +479,14 @@ class MediaMessageView: UIView, OWSAudioAttachmentPlayerDelegate {
         videoPlayer?.stop()
         videoPlayer?.view.removeFromSuperview()
         videoPlayer = nil
-        ViewControllerUtils.setAudioIgnoresHardwareMuteSwitch(false)
+        OWSAudioAttachmentPlayer.setAudioIgnoresHardwareMuteSwitch(false)
+    }
+}
+
+extension UIImage {
+    public convenience init?(named: String, inBundleForClass someClass: AnyClass) {
```

Comment:
I've just dropped this here for now, but I imagine there may be other resources that want to use this in the SAE. LMK if you have a good/ergonomic approach for having assets potentially distributed across frameworks.

Reply:
* Move to UIImage+OWS?  
* I think the other resources are certs, fonts (and localizable.strings which we don't need to worry about).

Reply:
Cert loading is already working fine - but I haven't looked at why.

Reply:
Based on our discussion, I've removed this method altogether since we're duplicating image assets to both bundles.